report/report-generator.js

```diff
@@ -102,13 +108,57 @@ class ReportGenerator {
     return '<script src="/pages/scripts/lighthouse-report.js"></script>';
   }
 
+  /**
+   * Refactors the PWA audits into their respective tech categories, i.e. offline, manifest, etc
+   * because the report itself supports viewing them by user feature (default), or by category.
+   */
+  _createPWAAuditsByCategory(aggregations) {
+    const items = {};
+
+    aggregations.forEach(aggregation => {
+      // We only regroup the PWA aggregations around so ignore any
```

Comment:
still don't understand the use of `around` here :) Seems to still make sense without it, so maybe just drop it?